mutatest/run.py

```diff
@@ -382,9 +373,16 @@ def create_mutation_run_trial(
 
     LOGGER.debug("Running trial for %s", mutation_op)
     mutant = genome.mutate(target_idx, mutation_op, write_cache=True)
-    mutant_trial = subprocess.run(
-        test_cmds, capture_output=capture_output(LOGGER.getEffectiveLevel())
-    )
+    try:
```

Comment:
@EvanKepner This is the crux of the changes: We pass the subprocess a timeout (which is a constant factor (say 5) times the expected execution time; the expected execution time is determined by a preliminary run of the test suite.

Reply:
This is a great feature add, thanks for doing the investigative work to put it together in a streamlined way.